packages/clerk-js/src/ui/components/Task/Task.tsx

```diff
@@ -0,0 +1,30 @@
+import { useSessionContext } from '@clerk/shared/react/index';
+import type { SessionTask } from '@clerk/types';
+import type { ComponentType } from 'react';
+
+import { OrganizationListContext } from '../../contexts';
+import { OrganizationList } from '../OrganizationList';
+
+const TaskRegistry: Record<SessionTask['key'], ComponentType> = {
+  org: () => (
+    <OrganizationListContext.Provider value={{ componentName: 'OrganizationList', hidePersonal: true }}>
```

Comment:
`hidePersonal` is being directly provided here, but this is going to be updated in a next PR.

We're going to include a property on `Enviroment` to indicate whether an after-auth task is enabled for instance. 

For the use case of forcing to select an org, this is going to be checked once the environment loads, and the org components are going to have `hidePersonal: true` by default.